tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -229,15 +229,30 @@ def get_binary_tensors(binary_paths, info, batch_sizes):
         current_batch_size = batch_sizes[shape_id]
         for b in range(current_batch_size):
             binary_index %= num_binaries
-            binary_filename = binary_paths[binary_index]
-            logger.info("Prepare binary file " + binary_filename)
-
-            binary_file_size = os.path.getsize(binary_filename)
-            blob_size = dtype.itemsize * int(np.prod(shape))
-            if blob_size != binary_file_size:
+            binary_filename:str = binary_paths[binary_index]
+            
+            extension = binary_filename.lower().split('.')[-1]
+            if extension == "npy":
+                logger.info("Prepare numpy file " + binary_filename)
+                numpy_arr:np.ndarray = np.load(binary_filename)
+                if list(numpy_arr.shape) != shape:
+                    raise Exception(
+                        f"Numpy array shape mismatch. File {binary_filename} has shape: {numpy_arr.shape}, expected: {shape}")
+                if numpy_arr.dtype != dtype:
+                    raise Exception(
+                        f"Numpy array is of {numpy_arr.dtype} format, which does not match input type. File {binary_filename}")
+                binaries[b] = np.reshape(numpy_arr, shape)
```

Comment:
Why do you reshape `numpy_arr`? Earlier you verified `list(numpy_arr.shape) == shape`

Reply:
Right, removed!